location_scrapbook/app/src/main/java/erica/locationscrapbook/MapsActivity.java

```diff
@@ -183,14 +178,17 @@ public void onClick(View view) {
                 input.setPositiveButton("Done", new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int id) {
                         if (useCurrent.isChecked()) {
-                            Marker currentLoc = mMap.addMarker(new MarkerOptions().position(current_location).title("Current Location").snippet("Description"));
+                            MarkerOptions newMarkerOpts = new MarkerOptions()
+                                    .position(current_location)
+                                    .title("Current Location")
+                                    .snippet("Description");
```

Comment:
I separated this line into multiple lines to make it more readable.  In general, when you find a line being really long, try to break it up into multiple lines.